charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -317,6 +325,15 @@ vmcluster:
   enabled: false
   # -- VMCluster annotations
   annotations: {}
+  # -- Custom VMAuth config, url_prefix requires only path, which will be appended to a select and insert base URL.
+  # To disable auth for vmselect or vminsert empty list for component config `vmcluster.vmauth.<component>: []`
+  vmauth:
+    vmselect:
+      - src_paths:
+          - /select/.*
+        url_prefix:
+          - /
+    vminsert: []
```

Comment:
I'd suggest enabling both routes by default in order to avoid confusion. 
There is already a comment clarifying how to disable routing for some of the components.